### PR TITLE
[css-flexbox] Inheritance, initial values

### PR DESCRIPTION
**Pull request by ewilligers**
Test that properties do not inherit.
Test that they have the expected initial values.
https://drafts.csswg.org/css-flexbox-1/#property-index

### CODE REVIEW

**Comment by ewilligers** [comment]
Note that 3 initial value tests fail as the spec is inconsistent with browsers and CSS Box Alignment.
https://github.com/w3c/csswg-drafts/issues/3636

**Comment by ewilligers** [comment]
I have update the initial value tests so they pass.

@fantasai please review.

**Comment by ewilligers** [comment]
@mrego  Please review.

File: css/css-flexbox/inheritance.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+</head>
+<body>
+<div id="container">
+  <div id="target"></div>
+</div>
+<style>
+  #container, #target {
+    display: flex;
+  }
+</style>
```
Comment (mrego) [inline css/css-flexbox/inheritance.html]: Nit: Kind of weird place for the `<style>` tag, I'd put it in the `<head>`.

**Review by mrego — APPROVED** [review]
LGTM.

**Pull request merged**